dotCMS/src/integration-test/java/com/dotmarketing/cms/urlmap/URLMapAPIImplTest.java

```diff
@@ -25,22 +28,28 @@
 import com.dotmarketing.util.PageMode;
 import com.liferay.portal.model.User;
 
+import java.util.Calendar;
+import java.util.Date;
 import java.util.List;
 import java.util.Optional;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
 public class URLMapAPIImplTest {
     public static final String TEST_PATTERN = "/testpattern";
     private static User systemUser;
     private static Host host;
 
     private URLMapAPIImpl urlMapAPI;
     private static HTMLPageAsset detailPage;
+    private static HttpSession session;
 
-    @BeforeClass
-    public static void prepare() throws Exception {
+    @Before
+    public void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4740221)